backend/apps/beaver_api/src/language_api/views.py

```diff
@@ -0,0 +1,9 @@
+from language_api.models import Language
+from language_api.serializers import LanguageSerializer
+
+from rest_framework.generics import ListAPIView
+
+
+class ListLanguageView(ListAPIView):
+    serializer_class = LanguageSerializer
+    queryset = Language.objects.all()
```

Comment:
No pagination specified because we don't expect many objects?

Reply:
That was the idea. But Now I think you are right and we can make it consistent with the rest of the APIs

Reply:
Oh, it was already there because I turned on default paginations in settings. But just for consistency, I made it explicit